OpsCenter_6.8_Release_Notes.md

```diff
@@ -1,5 +1,22 @@
 # Release notes for OpsCenter
 
+# Release Notes for OpsCenter 6.8.45
+17th June,2025
+
+## Core
+* Added the {{node_ips}} option to monitor nodes for events. Setting this field limits alerts monitoring to the nodes specified in the list. For example: ["127.0.0.1","127.0.0.2"]. (OPSC-17644)
```

Comment:
I'm still looking for it to be clarified that it is "alert" rather than "event" configuration.